File: Atividades A1/Exercicio 4/GO/ParesImparesEx4/main.go
package main

import "fmt"

func main() {
	var numbers [6]int
	var sumEven int
	var countOdd int
	fmt.Println("Digite 6 numeros inteiros:")
	// Receber os numeros do usuario
	for i := 0; i < 6; i++ {
		fmt.Scan(&numbers[i])
	}
	fmt.Print("Numeros pares digitados: ")
	for _, num := range numbers {
		if num%2 == 0 {
			fmt.Print(num, " ")
			sumEven += num
		}
	}
	fmt.Println()

	fmt.Println("Soma dos numeros pares:", sumEven)

	fmt.Print("Numeros impares digitados: ")
	for _, num := range numbers {
		if num%2 != 0 {
			fmt.Print(num, " ")
			countOdd++
		}
	}
	fmt.Print("\nQuantidade de numeros impares: ", countOdd)

}

File: Atividades A1/Exercicio 4/Java/ParesImparesEx4/src/Main.java
// Faça um programa em Java que receba 6 numeros inteiros e mostre
// - Os numeros pares digitados;
// - A soma dos numeros pares digitados;
// - Os numeros impares digitados;
// - A quantidade de numeros impares;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int[] numbers = new int[6];
        int sumEven = 0;
        int countOdd = 0;

        System.out.println("Digite 6 numeros inteiros: ");

        // Receba os numeros do usuário
        for(int i = 0;i < 6;i++) numbers[i] = scanner.nextInt();

        System.out.println("Numeros pares digitados: ");

        for(int num : numbers) {
            if (num % 2 == 0) {
                System.out.println(num + " ");
                sumEven += num;
            }
        }
        System.out.println("Soma dos valores pares: "+ sumEven);
        System.out.println("Numeros impares digitados: ");
        for(int num : numbers) {
            if (num % 2 != 0) {
                System.out.println(num + " ");
                countOdd += 1;
            }
        }
        System.out.println("Quantidade de numeros impares: " + countOdd);
    }
}